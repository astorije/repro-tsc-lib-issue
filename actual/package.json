{
  "name": "actual",
  "private": true,
  "scripts": {
    "build": "tsc"
  },
  "devDependencies": {
    "@types/node": "13.11.0",
    "typescript": "3.8.3"
  }
}
