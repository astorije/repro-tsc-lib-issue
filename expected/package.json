{
  "name": "expected",
  "private": true,
  "scripts": {
    "build": "tsc"
  },
  "devDependencies": {
    "typescript": "3.8.3"
  }
}
